modules/test/playwright/pages/commerce/commerce-channel-web/commerceAdminChannelDetailsPage.ts

```diff
@@ -327,6 +337,109 @@ export class CommerceAdminChannelDetailsPage {
 		await waitForAlert(await this.sidePanelNestedFrame('Shipping Methods'));
 	}
 
+	async addFixedTaxRate(amount: string, name: string) {
+		const tableName = 'Tax Calculations';
+
+		await (
+			await this.generalCommerceAdminChannelTableLink('Fixed Tax Rate')
+		).click();
+
+		await expect(
+			(await this.sidePanelFrame(tableName)).getByLabel('Active')
+		).toBeVisible();
+
+		await (await this.sidePanelFrame(tableName))
+			.getByLabel('Active')
+			.check();
+		await (await this.frameSaveButton(false, tableName)).click();
+
+		await (await this.taxRatesTab(tableName)).click();
+		await (await this.sidePanelFrame(tableName))
+			.getByTitle('Add Tax Rate')
+			.click();
+
+		await expect(
+			(await this.addTaxRateFrame()).getByText('Tax Category')
+		).toBeVisible();
+
+		await (await this.addTaxRateFrame())
+			.getByLabel('Tax Category')
+			.selectOption(name);
+		await (await this.addTaxRateFrame()).getByLabel('Amount').fill(amount);
+		await (await this.addTaxRateFrame())
+			.getByRole('button', {name: 'Submit'})
+			.click();
+
+		await this.page.reload();
+
+		await expect(
+			await this.generalCommerceAdminChannelTableLink('Fixed Tax Rate')
+		).toBeVisible();
+	}
+
+	async editFixedTaxRate(newAmount: string, name: string) {
+		const tableName = 'Tax Calculations';
+
+		await (
+			await this.generalCommerceAdminChannelTableLink('Fixed Tax Rate')
+		).click();
+		await (await this.taxRatesTab(tableName)).click();
+
+		await (await this.getRowByTextFromSidePanelTable(tableName, name))
+			.locator('.dnd-td')
+			.nth(2)
+			.getByRole('button', {name: 'Actions'})
```

Comment:
@smottal yeah, .locator('.dnd-td').nth(2) totally not needed, that's just redundant. On the other hand, if I create a locator for the rest, based on that I would need to do the same for a lot of other stuff. In that case, I think 
`(await this.getRowByTextFromSidePanelTable(tableName, name))`
will loose it's purpose because then I could just "burn in" the table's name.
What do you think?
Keep it dynamic like this or create a burned in version for everything with tablename "Tax Calculations"?

Reply:
The other stuff, I mean these:
```
await (await this.getRowByTextFromSidePanelTable(tableName, name))
			.getByRole('button', {name: 'Actions'})
			.click();

		await (await this.sidePanelFrame(tableName))
			.getByRole('menuitem', {name: 'Edit'})
			.click();

		await (await this.sidePanelNestedFrame(tableName))
			.getByLabel('Amount')
			.fill(newAmount);
```